share/rocm/cmake/ROCMClangTidy.cmake

```diff
@@ -197,7 +197,8 @@ function(rocm_clang_tidy_check TARGET)
                     set(CLANG_TIDY_COMMAND_LIST \"${CLANG_TIDY_COMMAND}\")
                     set(GH_ANNOTATIONS ${ROCM_ENABLE_GH_ANNOTATIONS})
                     execute_process(
-                        COMMAND ${CMAKE_COMMAND} --build ${CMAKE_CURRENT_BINARY_DIR} --target ${BASE_SOURCE}.i
+                        COMMAND
+                            ${CMAKE_COMMAND} --build ${CMAKE_CURRENT_BINARY_DIR} --target ${BASE_SOURCE}.i -e COLOR=OFF
```

Comment:
Where did you find documentation on the `-e COLOR=OFF` flag? And does this work with cmake 3.16? I'll try to run this locally.

Unfortunately, I dont think we have tests for tidy.

Reply:
The trailing options are forwarded to make where `-e` is used for environment overrides.